pkg/snapshot/synology.go

```diff
@@ -67,7 +67,7 @@ func GetSynology() (*Synology, error) { //nolint:cyclop
 			syn.Build = strings.Trim(lsplit[1], "\n\"")
 		case "manager":
 			syn.Manager = strings.Trim(lsplit[1], "\n\"")
-		case "vender":
```

Comment:
typo or is synology's payload misspelled?

Reply:
you better ask nit what's in his syno file.